### PR TITLE
Upgrade node-sass, test on node v6

### PR DESCRIPTION
**Pull request by daniel-ac-martin**
- Fixes the nastiness with pinning node-sass.
- Tests on node v6 instead of v5.

### CODE REVIEW

**Comment by daniel-ac-martin** [comment]
Rebased after #64.

File: package.json [hunk 1]
```diff
@@ -44,7 +44,7 @@
     "jquery": "^2.1.4",
     "moment": "^2.10.3",
     "moment-business": "^2.0.0",
-    "node-sass": "3.4.2",
+    "node-sass": "^3.7.0",
```
Comment (JoeChapman) [inline package.json]: If we are using hat character, we could just specify "3.0.0" to get latest minor or patch version?

Comment (daniel-ac-martin) [inline package.json]: There's a minimum minor version we need for node v6 support I believe.

**Comment by easternbloc** [comment]
LGTM

**Comment by JoeChapman** [comment]
LGTM

**Pull request merged**